spec.md

```diff
@@ -0,0 +1,238 @@
+  - [Artifact Publisher Manifest Example](#artifact-publisher-manifest-type-example) as a [Well Known Type][def-well-known-types]
+
+## Scope
+
+The scope of Artifacts v1.0 is based on what exists in 1.0 of [manifest][image-manifest].
+
+This includes individual artifacts that use [OCI manifest][image-manifest] including [OCI Image][image-spec], [Helm][helm] and [Singularity][singularity].
+
+### Future Scope
+
+Future versions of artifacts will support collections of artifacts using [OCI Index][image-index].
+
+To support artifacts that represent a collection of other artifacts, a means to identify an index is a type of artifact will be required.
+
+## Defining OCI Artifact Types
+
+As registries become content addressable distribution points, tools that pull artifacts must know if they can operate on the artifact. Artifact types are equivalent to file extensions. When users open files, the host operating system typically launches the appropriate program. When users open a file, from within a program, the open dialog filters to the supported types. When search or security software scan the contents of a storage solution, the software must to know how to process the different types of content. When users view the contents of a storage solution, they see the textual and visual indications of the type. OCI Artifacts provides these core capabilities to [OCI distribution spec][distribution-spec] based registries.
+
+Authoring an OCI Artifacts involves the following steps:
+
+- Define a unique type
+- Define the format for other tools to operate upon the type
+- Define human elements, such as an icon and localized string to be displayed to users
+- Optionally, publish as a [well-known type][def-well-known-types] for registries to consume
+
+Defining a unique type, string and logo; artifacts can be represented as the following:
+
+|Icon|Artifact|`config.mediaType`|
+|-|-|-|
+|<img src="https://github.com/opencontainers/artwork/blob/master/oci/icon/color/oci-icon-color.png?raw=true" width=30x>|[OCI Image][image-spec]|`application/vnd.oci.image.config.v1+json`|
+|<img src="https://github.com/helm/helm-www/blob/master/themes/helm/static/img/apple-touch-icon.png?raw=true" width=30x>|[Helm Chart](https://helm.sh)|`application/vnd.cncf.helm.chart.config.v1+json`|
+|<img src="https://github.com/sylabs/singularity/blob/master/docs/logos/singularity_v3.png?raw=true" width=30x>|[Singularity][singularity], by [Sylabs][sylabs]|`application/vnd.sylabs.sif.config.v1+json`|
+
+## Defining a Unique Artifact Type
+
+Defining the unique type involves uniqueness for computer processing, and uniqueness for humans.
+
+For computer processing, artifacts are defined by setting `manifest.config.mediaType` to a globally unique value.
+
+> **Note:** The `config.mediaType` of `application/vnd.oci.image.config.v1+json` is reserved for artifacts intended to be instanced by [docker][docker] and [containerd][containerd].  
```

Comment:
There are several alternative OCI image toolchains, including containers/image.  Can we just link into opencontainers/image-spec for the semantics of this type and say "for OCI images"?

Reply:
Please see new rewording. I'm trying to make sure readers understand oci-image isn't a generic collection of things. It's a very specific artifact type. I did add toolchains, but didn't call out containers/image as I wanted to avoid being asked to add all the other ones. That said, there's likely a place for related tool chains for each artifact type, such as [implementations.md](https://github.com/opencontainers/image-spec/blob/master/implementations.md)